conf/messages.en

```diff
@@ -465,26 +465,58 @@ pdf.ruling.aboutDecision.text=I have made this decision under section 24 of The
 pdf.ruling.samples=Samples
 pdf.ruling.keepSamples=Your samples have been kept by this office.
 pdf.ruling.collectSamples=Samples that are not collected or that do not have return postage will be destroyed after 60 days from the date of your ruling email. If you would like to collect your samples (or arrange for courier collection) then email <strong>{0}</strong> giving 48 hours notice.
-pdf.ruling.collectSamples.quote=You must quote the ATaR reference number <strong>{0}</strong> and give the time that the samples are being collected.
 pdf.ruling.coverLetter.title=Advance Tariff Ruling cover letter
 pdf.ruling.certificate.title=Advance Tariff Ruling Certificate
 pdf.ruling.certificate.issuedBy=Issued by: HMRC Tariff Classification Team
 pdf.ruling.certificate.rulingFor=Advance Tariff Ruling (ATaR) for {0}
+
 pdf.ruling.section.ifYouDisagree.title=What to do if you disagree with this decision
 pdf.ruling.section.ifYouDisagree=If you disagree with this decision, you can either:
 pdf.ruling.section.ifYouDisagree.1=ask for a review with HMRC
 pdf.ruling.section.ifYouDisagree.2=appeal to an independent tribunal
-pdf.ruling.section.ifYouDisagree.cannot=You cannot ask for a review with HMRC and an appeal with an independent tribunal at the same time. However, you can appeal to an independent tribunal if you disagree with the outcome of your review.
-pdf.ruling.section.ifYouDisagree.moreInfo=To find more information about reviews and appeals you can
-pdf.ruling.section.ifYouDisagree.read=read the HMRC1 document
-pdf.ruling.section.ifYouDisagree.phone=or phone 0300 200 3700.
+pdf.ruling.section.ifYouDisagree.within=You must do so within 30 days of the date of this letter.
+pdf.ruling.section.ifYouDisagree.cannot=You cannot ask for a review with HMRC and an appeal with an independent tribunal at the same time. However, you can appeal to an independent tribunal if you disagree with the outcome of your review. If you first appeal to a tribunal, a review cannot be offered.
+
 pdf.ruling.section.askForReview.title=Asking for a review with HMRC
-pdf.ruling.section.askForReview.email=Email <strong>{0}</strong> asking for a review and give your reasons within 30 days of receiving your ruling email.
-pdf.ruling.section.ruling.details={0} details
```

Comment:
This was mistakenly removed so the content is not rendered properly. Please add this back, thanks